todos/src/App.js

```diff
@@ -1,23 +1,37 @@
-import React from "react";
+import React, { useEffect, Fragment } from "react";
 import "App.css";
 import Navbar from "components/Navbar";
 import Todos from "components/Todos";
-import { Provider } from "react-redux";
-import store from "store";
 import TopContent from "components/TopContent";
 import { ToastContainer } from "react-toastify";
+import { useDispatch } from "react-redux";
+import { fetchInitialData } from "actions/index";
+import { useSelector } from "react-redux";
+import SplashScreen from "components/SplashScreen";
 
 function App() {
-  return (
-    <React.Fragment>
-      <Provider store={store}>
+  const splashLoadingState = useSelector(
+    (state) => state.loadingReducers.splashLoadingState
+  );
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchInitialData());
+  }, []);
+
+  return splashLoadingState ? (
+    <SplashScreen />
+  ) : (
+    <>
+      <Fragment>
```

Comment:
`<>` and `<Fragment />` are equivalent to each other. One of those is unnecessary here.